src/gleam/string.gleam

```diff
@@ -148,27 +203,34 @@ pub fn contains(does haystack: String, contain needle: String) -> Bool {
     erl_contains(haystack, needle)
 }
 
-// TODO
-// TODO: Not sure about the name and labels here
 /// See if the second string starts with the first one.
 ///
 /// ## Examples
 /// ```gleam
 /// starts_with(does: "theory", start_with: "ory") == False
 /// ```
 ///
-// pub fn starts_with(does string: String, start_with prefix: String) -> String {}
+external fn erl_string_starts_with(string: String, prefix: String) -> Bool =
+  "gleam_stdlib" "string_starts_with"
+
+pub fn starts_with(does does: String, start_with start_with: String) -> Bool {
+    erl_string_starts_with(does, start_with)
+}
 
-// TODO
-// TODO: Not sure about the name and labels here
 /// See if the second string ends with the first one.
 ///
 /// ## Examples
 /// ```gleam
 /// ends_with(does: "theory", end_with: "ory") == True
 /// ```
 ///
-// pub fn ends_with(does string: String, end_with suffix: String) -> String {}
+external fn erl_string_ends_with(string: String, suffix: String) -> Bool =
+  "gleam_stdlib" "string_ends_with"
```

Comment:
We can export this function directly rather than wrapping it with another.

Reply:
The parser doesn't allow named paramaeters on external functions:
```
===> Compiling gleam_stdlib

error: Syntax error

     âââ string.gleam:226:32 âââ
     â
 226 â pub external fn ends_with(does string: String, end_with suffix: String) -> Bool =
     â                                ^^^^^^ Unexpected token
     â

Expected one of ")", ",", ".", ":"
```

Reply:
Ah! I haven't documented how to use labelled arguments with external functions. I'll have to fix that.

It should look like this: https://github.com/gleam-lang/stdlib/commit/094cf5f6d6f9dac61806f883d3677d357d3a7f38

Reply:
sweet! 👍